pages/components/navigation.html

```diff
@@ -107,7 +107,25 @@ <h1 clss="page-title">navigation</h1>
 
   <main class="main-content">
 
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+  <nav class="nav nav-site bg-primary">
+    <h2><a class="heading list-heading" href="/">Ira</a></h2>
+    <ul class = "list list-inline">
+      <li class="list-item"> <a href=/>About </a> </li>
+      <li class="list-item"> <a href=/>Documentation </a> </li> 
+      <li class="list-item"> <a href=/>Installation </a> </li> 
+      <li class="list-item"> <a href="https://github.com/HarshnaNHaswani/Ira">GitHub</a></li>
+    </ul>
+  </nav>
+
+  <nav class="nav nav-scrolling bg-secondary">
+    <ul class = "list list-inline">
+      <li class="list-item"> <a href=/>About </a> </li>
+      <li class="list-item"> <a href=/>Documentation </a> </li> 
+      <li class="list-item"> <a href=/>Installation </a> </li> 
+      <li class="list-item"> <a href="https://github.com/HarshnaNHaswani/Ira">GitHub</a></li>
+    </ul>
+  </nav>
+
```

Comment:
why 2 navs? i can't understand as you have not added any screenshots to see.

Reply:
Is second nav for responsiveness initially you are hiding something like this

Reply:
Forgot to add, please check it now